def log_exception(*args, **kwargs):
    pass
